packages/@aws-cdk/region-info/build-tools/generate-static-data.ts

```diff
@@ -61,6 +62,11 @@ async function main(): Promise<void> {
 
     registerFact(region, 'APPMESH_ECR_ACCOUNT', APPMESH_ECR_ACCOUNTS[region]);
 
+    const firehoseCidrBlock = FIREHOSE_CIDR_BLOCKS[region];
+    if (firehoseCidrBlock) {
+      registerFact(region, 'FIREHOSE_CIDR_BLOCK', `${FIREHOSE_CIDR_BLOCKS[region]}/27`);
```

Comment:
All of the regions use port /27 except Africa cape town. Do we think that's a typo in the docs? https://docs.aws.amazon.com/firehose/latest/dev/controlling-access.html#using-iam-rs-vpc